### ISSUE TITLE
Using the "editMessageText" method

### ISSUE THREAD

**Issue by hikuikuma** [scraton/node-red-contrib-telegrambot-home#34]
I wanted to use the "editMessageText" method via the "payload" node to modify a message while waiting for the request to be processed but the node returns "empty arguments after parsing payload".

Here is what I send to the node :

```
msg.method = 'editMessageText';
msg.payload = {
    'text':'<i>Patientez...</i>',
    'message_id':msg.telegram.message_id,
    'chat_id':msg.telegram.chat.id,
    'parse_mode':'HTML'
}
```

**Comment by scraton** [comment]
Thanks for the report! Looks like the payload node doesn't support that method yet. And there appear to be a number of issues with related commands.

I've pushed an update that should fix this. I tested your example function code and it passed on my end.

Could you try updating to 0.7 and see if it works for you?

**Comment by hikuikuma** [comment]
Thank you very much for the reactivity, the fix in 0.7.0 works very well on my side! Good dev to you!
